notebooks/official/experiments/build_model_experimentation_lineage_with_prebuild_code.ipynb

```diff
@@ -174,9 +174,8 @@
         "if IS_WORKBENCH_NOTEBOOK:\n",
```

Comment:
<p>Is there a reason not to clean up the bucket by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>That's a good question. I think it is as a precaution. Andy suggested to set it as False. </p>